dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -38,7 +38,7 @@ class ScriptsControllerTest < ActionController::TestCase
   end
 
   test "should get show of hoc" do
-    get :show, id: Script::HOC_ID
```

Comment:
should we also add some tests of the redirects? (test/integrations/redirects_test.rb)